resume.md

```diff
@@ -18,42 +18,42 @@ Professional Experience
 **Honest Food Company([website](https://honestfoodcompany.de))**
 
 * *Software Engineer* | December 2019 - Present
-    * Project Lead on customer after order feedback
-    * Serverless design and architecture with Google Cloud/Terraform
-    * NodeJS, Typescript heavy codebases
+    * Project Lead on customer after order feedback application
+    * Utilized serverless design and architecture with Google Cloud/Terraform
+    * Coding languagages: NodeJS and Typescript heavy
 
 **Sightbox([sightbox](https://sightbox.com))**
 
 * *Software Engineer* | October 2018 - December 2019
-    * Management of the supply chain infrastructure for contact lense ordering and delivery.
```

Comment:
Oh I also removed some periods at the end of bullet points. Generally bullets don't need them but I see a mix so just keep it consistent. I tried to write this as a second commit but that didn't work. It's been a while since I used Github. :P